daemon/listeners/group_unix.go

```diff
@@ -6,25 +6,15 @@ import (
 	"fmt"
 	"strconv"
 
-	"github.com/opencontainers/runc/libcontainer/user"
-	"github.com/pkg/errors"
```

Comment:
Ah.. this broke master in combination with https://github.com/moby/moby/pull/38316 fix upcoming

Reply:
Fix https://github.com/moby/moby/pull/38360 👍